chat/src/main/java/com/github/twitch4j/chat/TwitchChatBuilder.java

```diff
@@ -257,6 +257,17 @@ public class TwitchChatBuilder {
     @With
     private IBackoffStrategy connectionBackoffStrategy = null;
 
+    /**
+     * Whether the {@link #getChatAccount()} should be validated on reconnection failures.
+     * <p>
+     * If enabled and the token has expired, chat will connect in read-only mode instead.
+     * <p>
+     * If the network connection is too slow, you may want to disable this setting to avoid
+     * disconnects while waiting for the result of the validate endpoint.
+     */
+    @With
+    private boolean validateReconnectToken = true;
```

Comment:
any better naming ideas?

Reply:
as we provide the token in/as `chatAccount` how about `verifyChatAccountOnReconnect`?